roles/openshift_master_upgrade/handlers/main.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
It really seems like we need an openshift_master_common that can hold things that are not openshift_master_facts?